src/v/redpanda/admin/server.cc

```diff
@@ -1860,6 +1863,16 @@ admin_server::patch_cluster_config_handler(
                       yaml_name,
                       property.format_raw(yaml_value),
                       validation_err.value().error_message());
+                } else if (auto restricted_err = property.check_restricted(val);
+                           restricted_err.has_value() && should_sanction) {
+                    errors[yaml_name] = restricted_err.value().error_message();
+                    vlog(
+                      adminlog.warn,
+                      "Rejected config due to invalid enterprise license {}: "
+                      "'{}' ({})",
```

Comment:
nitpick: Given that the error message is of the form "'{}' is restricted to enterprise licensed clusters", is the end result a touch repetitive?

Reply:
yeah probably. will revisit in a followup